trillian/ctfe/cache/cache.go

```diff
@@ -0,0 +1,21 @@
+package cache
+
+import "context"
+
+// NonLeafCertificateChainCache is an interface which allows CTFE binaries to use different cache implementations for non-leaf certificate chains.
+type NonLeafCertificateChainCache interface {
+	// IsLazyLoading returns whether lazy loading is enabled.
+	IsLazyLoading() bool
+
+	// Contains returns whether the hash exists in the cache.
+	Contains(ctx context.Context, hash string) (bool, error)
+
+	// Get returns the non-leaf certificate chain by the hash.
```

Comment:
s/by the/under

Reply:
Updated to `// Get returns the non-leaf certificate chain associated with the provided hash.`.